src/NServiceBus.TransportTests/When_scope_dispose_throws.cs

```diff
@@ -39,31 +40,31 @@ public class When_scope_dispose_throws : NServiceBusTransportTest
             // dispose throws the message might be picked up before the counter is incremented
             Assert.LessOrEqual(1, errorContext.ImmediateProcessingFailures);
         }
-    }
 
-    class EnlistmentWhichFailesDuringPrepare : IEnlistmentNotification
-    {
-        public static readonly Guid Id = Guid.NewGuid();
-
-        public void Prepare(PreparingEnlistment preparingEnlistment)
+        class EnlistmentWhichFailesDuringPrepare : IEnlistmentNotification
         {
-            // fail during prepare, this will cause scope.Dispose to throw
-            preparingEnlistment.ForceRollback();
-        }
+            public static readonly Guid Id = Guid.NewGuid();
 
-        public void Commit(Enlistment enlistment)
-        {
-            enlistment.Done();
-        }
+            public void Prepare(PreparingEnlistment preparingEnlistment)
+            {
+                // fail during prepare, this will cause scope.Complete to throw
```

Comment:
Given the correction to this comment, is the name of the test class, `When_scope_dispose_throws`, incorrect also?